servers/src/grin/sync.rs

```diff
@@ -199,8 +199,8 @@ fn do_header_sync(
 ) {
 	let status = sync_state.status();
 
-	let update_sync_state = match status {
-		SyncStatus::TxHashsetDownload => false,
+	let doit = match status {
```

Comment:
A nitpick but something like `enable_header_sync` would be a lot more explicit.

Reply:
forgive my english~  updated.